contracts/modules/staking/PooledStaking.sol

```diff
@@ -1474,7 +1474,6 @@ contract PooledStaking is IPooledStaking, MasterAware {
     uint deposit = stakers[msg.sender].deposit;
     stakers[msg.sender].deposit = 0;
     token.approve(address(tokenController), deposit);
-    uint stakePositionNFTId = stakingPool.deposit(deposit, groupId, 0);
-    stakingPool.safeTransferFrom(address(this), msg.sender, stakePositionNFTId);
+    stakingPool.deposit(deposit, groupId, 0, msg.sender);
```

Comment:
Add manager to the createStakingPool call above this part.

Reply:
clarified offline